### PR TITLE
fix(keyExpectedValue): ansible-aws queries convert to a recommendation rather than a current status

### PR DESCRIPTION
**Pull request by liorj-orca**
Closes #

**Proposed Changes**
-
-
-

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/ansible/aws/iam_password_without_uppercase_letter/query.rego [hunk 1]
```diff
@@ -18,7 +18,7 @@ CxPolicy[result] {
 		"resourceName": task.name,
 		"searchKey": sprintf("name={{%s}}.{{%s}}.require_uppercase", [task.name, modules[m]]),
 		"issueType": "IncorrectValue",
-		"keyExpectedValue": "iam_password_policy.require_uppercase set and true",
+		"keyExpectedValue": "iam_password_policy.require_uppercase should be set and true",
```
Comment (rafaela-soares) [inline assets/queries/ansible/aws/iam_password_without_uppercase_letter/query.rego:21]: Can you improve the `keyActualValue` of this result, please?

```
"keyActualValue": "iam_password_policy.require_uppercase is undefined",
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]
Hello, @liorj-orca!

Thank you so much for another contribution!

Can you update your branch with the master, please?
I only have a minor suggestion:

File: assets/queries/ansible/aws/iam_password_without_uppercase_letter/query.rego (hunk 1 above)
Comment (liorj-orca) [reply assets/queries/ansible/aws/iam_password_without_uppercase_letter/query.rego:21]: Hi @rafaela-soares,
NP, 
will something like this be OK:
"keyActualValue": "iam_password_policy.require_uppercase is not set to true" or you have something else on your mind?

Comment (liorj-orca) [reply assets/queries/ansible/aws/iam_password_without_uppercase_letter/query.rego:21]: sure NP

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀

**Pull request merged**